NAMESPACE

```diff
@@ -17,3 +17,36 @@ exportMethods(getLinkDSLD)
 exportMethods(getLinkTTG)
 exportMethods(initialize)
 exportMethods(merge)
+<<<<<<< Updated upstream
+=======
+importFrom(assertthat,"TRUE")
+importFrom(assertthat,"for")
+importFrom(assertthat,"function")
+importFrom(assertthat,"if")
+importFrom(assertthat,A)
+importFrom(assertthat,Is)
```

Comment:
@charksong , Looks like something has gone wrong here. You've included the merge conflict  brackets (the `<<<<<<` and `=====` symbols)  and you also appear to have imported random words from the assertthat library that I'm pretty sure are not functions within it :)